hs/MakeElements.hs

```diff
@@ -35,6 +35,7 @@ jQuery = makeScript "https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.
 timetableLinks :: H.Html
 timetableLinks = concatHtml (map stylesheet ["//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css",
                                          "static/style/grid/timetable_styles.css",
+                                         "static/style/graph/graph_styles.css",
```

Comment:
@Ian-Stewart-Binks Since the modal styles have been added into the common styles, this line is no longer needed.